config/routes.rb

```diff
@@ -25,6 +25,7 @@
   match 'experimental/job_invocations_detail/:id', to: 'react#index', :via => [:get], as: 'new_job_invocation_detail'
   match 'job_invocations_detail/:id/host_invocation/:host_id', to: 'react#index', :via => [:get], as: 'new_job_invocation_detail_by_host'
   get 'show_template_invocation_by_host/:host_id/job_invocation/:id', to: 'template_invocations#show_template_invocation_by_host'
+  get 'job_invocations/:id/hosts', to: 'template_invocations#list_jobs_hosts'
```

Comment:
(not blocking, just two cents)
Is it just me or this is rather confusing? Why do we keep mention `job` but use `template` across at least this PR? I mean, I probably get that we want all the information during 1 call instead of multiple ones, but if we want Rails' way, should we rather be more explicit and place/use things where they are expected to be?..

Reply:
Thanks, moved